rotkehlchen/db/dbhandler.py

```diff
@@ -1650,6 +1650,19 @@ def set_rotkehlchen_premium(self, credentials: PremiumCredentials) -> None:
         # an empty last write ts in that case
         # self.update_last_write()
 
+    def del_rotkehlchen_premium(self) -> bool:
+        """Delete the rotki premium credentials in the DB for the logged-in user"""
+        try:
+            cursor = self.conn.cursor()
+            cursor.execute(
+                'DELETE FROM user_credentials WHERE name =?', ('rotkehlchen',),
```

Comment:
can you remove the space between `name` and `=`? Just for consistency with the rest of the file.

Reply:
Done.